README.md

```diff
@@ -177,6 +180,46 @@ By default:
     }
     ```
 
+#### Paginating non-scalar results
+
+When query doesn't returned scalars, specify `scalars=False` when invoking `paginate`:
```

Comment:
```suggestion
To paginate a query which doesn't return scalars, specify `scalars=False` when invoking `paginate`:
```

Also, is scalar a known/common enough concept, or should we perhaps link to associated docs here?